docs/cookbook/shop/cms.rst

```diff
@@ -0,0 +1,43 @@
+How to manage content in Sylius?
+=================================
+
+Why do you need content management system?
+------------------------------------------
+
+Content management is one of the most important business aspects of modern eCommerce apps.
+Providing store updates like new blog pages, banners and promotion images is responsible for building the conversion rate
+either for new and existing clients.
+
+Content management in Sylius
+----------------------------
+
+Default Sylius standard app does not come with a content management system. Don’t worry - our community has taken care of it.
+As Sylius does have an awesome dev oriented plugin environment, our friends from `BitBag <https://bitbag.shop>`_ decided to develop
```

Comment:
Why are you writing about yourself as "our friends"? :D We love you BitBag, but to remain politically correct I'd get rid of this expression in the docs. :P